### ISSUE TITLE
Doesn't work

### ISSUE THREAD

**Issue by johnathanschneider** [Shywim/github-repo-size#43]
Not much else to say. Sonoma 14.3.1, Firefox 123.0.1. No changes in the summary header of a github repo, even this one. i have no other extensions or userscripts enabled affecting github enabled.

Edit: I've noticed a large handful of extensions that affect github that do not work on my firefox instance - however, some extensions on AMO designed to alter Github _do_ work, so I'm not sure what the discrepancy there is.

**Comment by Deetss** [comment]
@johnnyschneider 

I've fixed this and it should work now. Go here for the updated version I forked from this repo.

https://addons.mozilla.org/en-US/firefox/addon/deetss-github-repo-size/

Here's the repository I have forked it to so you can check for any funniness if you don't trust me.
https://github.com/deetss/github-repo-size

P.S. I just now submitted the Addon so it may take a while to update or be publicly available. 

If you need the addon now clone my repo and run `npm run build`
You'll be able to upload the addon to Firefox on your own to start using it.

**Comment by ArianeBouchardConformit** [comment]
Thanks @Deetss . Your plugin works great. I wonder if there's a mechanism for you to request ownership to actually replace the old one.

**Comment by Deetss** [comment]
@ArianeBouchardConformit I don't believe there is. I wanted to make a PR on this repo but the owner hasn't touched this in 2 years so I opted to make my own. If at some point this becomes active again I will have him merge my fork in, but in the meantime feel free to use mine!
